archive/c/c-plus-plus/linear-search.CPP

```diff
@@ -0,0 +1,26 @@
+// C++ code to linearly search x in arr[]. If x 
```

Comment:
Can we remove this file for this pull request?

Reply:
Yes I will remove that.